components/style/values/computed/mod.rs

```diff
@@ -114,6 +114,76 @@ impl<T> ToComputedValue for T
     }
 }
 
+/// A computed `<angle>` value.
+#[derive(Clone, PartialEq, PartialOrd, Copy, Debug)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf, Deserialize, Serialize))]
+pub struct Angle {
```

Comment:
Hmm, this makes me realize that Gecko stores the specific angle unit in computed values of `linear-gradient()`, but Servo here would serialize them as radians.